packages/react-jsx-highcharts-gantt/src/components/Pathfinder/Pathfinder.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Pathfinder extends Component {
+
+  static propTypes = {
+    enabled: PropTypes.bool.isRequired,
+    dashStyle: PropTypes.string(),
+    lineWidth: PropTypes.number()
+  };
+
+  static defaultProps = {
+    enabled: true
+  };
+
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      rendered: false
+    };
+  }
+
+  render () {
+    // TODO: Howto?
```

Comment:
[This component](https://github.com/whawker/react-jsx-highcharts/blob/d3257a8ed0f30581ed90210b69fb8071596e629d/packages/react-jsx-highcharts/src/components/Legend/Legend.js) might help.

You need to [provide the chart](https://github.com/whawker/react-jsx-highcharts/blob/d3257a8ed0f30581ed90210b69fb8071596e629d/packages/react-jsx-highcharts/src/components/Legend/index.js#L3-L4) to the component

Then `chart.update` with new `pathfinder` config on [`componentDidMount`](https://github.com/whawker/react-jsx-highcharts/blob/d3257a8ed0f30581ed90210b69fb8071596e629d/packages/react-jsx-highcharts/src/components/Legend/Legend.js#L20-L22) and [`componentDidUpdate`](https://github.com/whawker/react-jsx-highcharts/blob/d3257a8ed0f30581ed90210b69fb8071596e629d/packages/react-jsx-highcharts/src/components/Legend/Legend.js#L26-L28). 

Finally, [disable](https://github.com/whawker/react-jsx-highcharts/blob/d3257a8ed0f30581ed90210b69fb8071596e629d/packages/react-jsx-highcharts/src/components/Legend/Legend.js#L33) the pathfinder on unmount.

Then component will have no children, so  you can just `return null` from render